### DISCUSSION TITLE
Better design for saving user data

### DISCUSSION THREAD

**Discussion by BeritJanssen** [UUDigitalHumanitieslab/I-analyzer#1578] [General]
We should investigate if we can do better with the way we save user data, especially for the scenario of federated logins.

**Comment by lukavdplas** [comment]
Can you describe what you would like to see improved?

**Comment by BeritJanssen** [comment]
Hashing email addresses, for instance.

**Comment by lukavdplas** [comment]
@BeritJanssen can we make this a discussion? It doesn't look like there is a concrete, workable plan for this. (Or if you have one, can you add it in the issue?)
